### ISSUE TITLE
[BUG] json_decode(): Argument #1 ($json) must be of type string, array given

### ISSUE THREAD

**Issue by ndberg** [coroo/nova-chartjs#169]
**Describe the bug**
Out of nowhere I get an error:
```php
json_decode(): Argument #1 ($json) must be of type string, array given
array:2 [▶
  "userId" => 1
  "exception" => TypeError {#1602▶
    #message: "json_decode(): Argument #1 ($json) must be of type string, array given"
    #code: 0
    #file: "/var/www/backend/vendor/coroowicaksono/chart-js-integration/src/api/TotalRecordsController.php"
    #line: 26
    trace: {▶}
  }
]
```

**To Reproduce**
1. Add a Chart
2. Add anything to the options array, like:
`->options(['uom' => 'day',])
`

As soon as any option is present, the error shows up. It looks like the NovaRequest $request->options[] changed from json to array? But couldn't find a change like this in the Nova releases. 

Does anybody else have this error?

**Dependencies Version:**
 - Laravel Version: 10.32.1
 - Nova Version: 4.31.3
 - Nova-ChartJS Version: 0.4.1

**Comment by bmellink** [comment]
I have the same issue. It seems laravel/nova has a breaking change between versions 4.29.5 and 4.29.6 that impacts nova-chartjs. Temporary work around is to change your composer.json and change the line that reads

`"laravel/nova": "~4",`

to 

`"laravel/nova": "4.29.5",`

This will force using the laravel/nova version that is still compatible with nova-chart-js. I hope it will be fixed soon in nova-chart-js.
The issue is in `TotalRecordsController.php` function `handle()` where `$request->options` used to be passed as a json string and with the new version of laravel/nova are passed as array, so all the `json_decode()` commands generate exceptions.

**Comment by ndberg** [comment]
You're right, it works with laravel/nova v4.29.5. 

Thanks for helping

**Comment by coroo** [comment]
Release new version 0.4.2. Hopefully can help to fix the error for laravel nova 4.31.x
https://github.com/coroo/nova-chartjs/releases/tag/v0.4.2
